lib/chutney/TorNet.py

```diff
@@ -767,32 +766,30 @@ def _genRouterKey(self):
             sys.exit(1)
         self._env['fingerprint'] = fingerprint
        
-
     def _setEd25519Id(self):
         """Read the ed25519 identity key for this router, and set up the 'ed25519-id' entry in the Environ"""
         datadir = self._env['dir']
         key_directory = os.path.join(datadir, 'keys', "ed25519_master_id_public_key")
         EXPECTED_ED25519_FILE_SIZE = 64
         CURRENT_FILE_SIZE = os.stat(key_directory).st_size
-        if os.path.exists(key_directory) == False:
+        if not os.path.exists(key_directory):
             raise ValueError("File does not exist")
```

Comment:
Some old tor versions don't have an `ed25519_master_id_public_key`. And I just remembered that we need to support some very old tor versions in chutney.

So we can't raise an error here. Instead, let's log a message, and return?
```suggestion
            print("File {} does not exist. Are you running a very old tor version?".format(key_directory))
            return
```